OpenSim/Simulation/Model/ContactGeometry.h

```diff
@@ -46,7 +46,7 @@ OpenSim_DECLARE_ABSTRACT_OBJECT(ContactGeometry, ModelComponent);
 // DATA
 //=============================================================================
 protected:
-    Body* _body;
+    SimTK::ReferencePtr<PhysicalFrame> _body;
```

Comment:
In BushingForce, `T = const PhysicalFrame`. Should that be the case here as well?